helpers/dragRangeSliderLowerBoundTo.ts

```diff
@@ -1,10 +1,10 @@
 declare namespace WebdriverIOAsync {
   interface Browser {
-    setRangeSliderLowerBound(value: number): Promise<number>;
+    dragRangeSliderLowerBoundTo(value: number): Promise<number>;
   }
 }
 
-browser.addCommand('setRangeSliderLowerBound', async (value: number) => {
+browser.addCommand('dragRangeSliderLowerBoundTo', async (value: number) => {
   const slider = await browser.$('.rheostat-horizontal');
   let lowerHandle = await browser.$('.ais-RangeSlider .rheostat-handle-lower');
```

Comment:
This is not going to work in all flavors because we don't use Rheostat in all of them.

Reply:
Then we'll need to update the helper to make it work with the other flavors too.

But we'll see this in detail when we'll integrate to e2e tests in the other flavors.